metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-env.xml

```diff
@@ -253,69 +253,295 @@
         </value-attributes>
     </property>
     <property>
-        <name>global-json</name>
-        <display-name>global.json template</display-name>
-        <description>This is the jinja template for global.json file</description>
-        <value>
-{
-"es.clustername": "{{ es_cluster_name }}",
-"es.ip": "{{ es_url }}",
-"es.date.format": "yyyy.MM.dd.HH",
-"parser.error.topic": "indexing"
-}
-        </value>
+        <name>es_date_format</name>
+        <description>Elasticsearch Date Format</description>
+        <value>yyyy.MM.dd.HH</value>
+        <display-name>Elasticsearch Date Format</display-name>
+    </property>
+    <property>
+        <name>parser_error_topic</name>
+        <description>Parser Error Topic</description>
+        <value>indexing</value>
+        <display-name>Parser Error Topic</display-name>
+    </property>
+    <property>
+        <name>enrichment_workers</name>
```

Comment:
This issue existed before this PR but I think it's annoying and would be happy to fix it.  I will have to reorganize metron-env.xml into component-level files so I can make the configuration dependencies more granular.  Will try to get this turned around quickly.